### DISCUSSION TITLE
Bracket Pair Colorizer

### DISCUSSION THREAD

**Discussion by slabach** [coder/code-server#2442] [Q&A]
I am having issues getting bracket pair colorizer (or 2) working. Has anyone else had issues with this and found a fix? 

Btw, I just found out about code-server and self hosting it yesterday and was sold! Completely changed my workflow. Thanks for all the work everyone has done on this!!

**Comment by kphanipavan** [comment]
I am having issues as well.
Both Bracket Pair Colorizer and 2 are not working.

**Comment by code-asher** [comment]
I forgot to update this discussion; this should be fixed in 3.8.0.

https://github.com/cdr/code-server/issues/2197
